monitoring/monit.py

```diff
@@ -86,8 +88,35 @@ def run_command(command):
         module.run_command('%s %s %s' % (MONIT, command, name), check_rc=True)
         return status()
 
-    process_status = status()
-    present = process_status != ''
+    def wait_for_monit_to_stop_pending(sleep_time=1):
```

Comment:
sure, I can absolutely up that. I'm a bit wary of 10s between checks though. The services I'm running usually go into start/stop state in about 2-3 seconds and waiting that long would seem excessive. 

Maybe look at a gradual back-off up to 10s?